tests/Nox.Types.Tests/Types/CountryCode2/CountryCode2TestsDataClass.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Collections;
 
-namespace Nox.Types.Tests.Types.CountryCode3;
```

Comment:
In other tests the namespace is Nox.Types.Tests.Types. Can we update this one as well?

Reply:
Almost there. The namespace is Nox.Types.Tests.Types.